### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by tomalec**
I'd like to propose few improvements to issue templates:

- Use headings in issues templates,
	instead of bold paragraphs.
	To match with PR template, to improve readability and accessibility.
- Add the "Actual behavior" section.
	I believe it's sometimes valuable to clearly state what currently happens, so the reviewer, would not have to get that from the description or test locally.
- Reorder sections,
	I think screenshots (and actual behavior) describes what you have immediately after you perform "steps to reproduce", so therefore should be stated there. Not after the expected behavior, which is not happening, and it's usually hard to take a screenshot of it.
- Move section descriptions to comments.
	This way the issuer could still keep them in place while filling the sections, answering those requests.
- Use ellipsis instead of three dots.
	To make it more readable, by fully sighted and visually impaired readers https://tosbourn.com/the-importance-of-using-the-ellipsis-character not to read it as "dot dot dot"

### Accessibility

-   [x] I've tested using only a keyboard (no mouse)
-   [x] I've tested using a screen reader
-   [n/a] All animations respect [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
-   [n/a] All text has [at least a 4.5 color contrast with its background](https://webaim.org/resources/contrastchecker/)

### Detailed test instructions:

-   Preview formatted template https://github.com/woocommerce/woocommerce-admin/blob/update/issue-templates/.github/ISSUE_TEMPLATE/BUG_REPORT.md
- 

> Please add a Changelog note

Given the change does not affect the code at all, I doubt it needs a changelog entry.

----

no changelog necessary as it is `.github` change only

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 1]
```diff
@@ -5,35 +5,39 @@ about: Create a report to help us improve
...
-2. Click on '....'
-3. Scroll down to '....'
+1. Go to '…'
+2. Click on '…'
+3. Scroll down to …'
 4. See error
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+### Actual behavior:
```
Comment (joshuatf) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:19]: This is the only piece of this PR I feel on the fence about.  I feel it's redundant when a bug is described above and more often than not we're left wondering what someone expected to happen.

**Review by joshuatf — COMMENTED** [review]
Thanks for these updates, @tomalec!  Great improvements all around; just left one comment about the "Actual behavior" section.

**Comment by tomalec** [comment]
I agree that is very basic cases it's more-or-less the same. 

When the problem is more complicated, I find it valuable to emphasize and summarize in short what happens.
So the description could be really elaborate, give the full context, background, conditions under which something happens.

See https://github.com/woocommerce/google-listings-and-ads/issues/652 for example.

So in my mental model, the information flow was like this:

Describe elaborately the background, the problem, and how to reproduce it, summarize shortly what currently happens, add a visual example. Then state what you expect to happen.

The person filing the issue could always remove that section if it's redundant. 

----

Personally, I was confused with "Screenshots" been placed after "Expected". Should the screenshot show the bug I found, or something I expect to happen? How could I make a screenshot of something that does not happen?

----

Or do you mean you would prefer the following?
1. description
2. steps to reproduce
3. screenshots
4. actual
5. expected


----

Naturally, If you find that still confusing and redundant, I'm happy with removing the "Actual" section.

**Comment by tomalec** [comment]
ping @joshuatf, as I'm not sure which way and if I should change something here.

**Review by joshuatf — APPROVED** [review]
Hey @tomalec, thanks for the ping; I hadn't seen your previous message.

While I think the issue description can usually cover actual behavior, I think you're also right that authors can easily remove sections that aren't applicable.

Thanks for taking the time to improve the template!  LGTM 🚢

**Pull request merged**